local-cli/desktop/runDesktop.js

```diff
@@ -71,8 +71,11 @@ function actuallyRun(args, reject) {
         appArgs.push('--host=' + args['host']);
       if (args['port'])
         appArgs.push('--port=' + args['port']);
-      if (args['executor'])
+      if (args['executor']) {
         appArgs.push('--executor=' + args['executor']);
+      } else {
+        appArgs.push('--executor=RemoteServerConnection');
```

Comment:
> do I understand correctly that all error messages that we were receiving in ./ubuntu-server.js output now will be shown in application output?
@vkjr , if you have start application as `react-native run-desktop` it will replace default connection(piped process) with RemoteServerConnection(as it referenced in the source code above). But, yes, if we start application directly, for example, from root directory of project it will be like: "./desktop/bin/ProjectBinary" ... it will use started with piped node process and all JS output will be redirected to the native app process output.

Reply:
@MaxRis, thanks, got it :)